src/Dofs/DofHandler.jl

```diff
@@ -938,10 +938,10 @@ function _evaluate_at_grid_nodes(dh::DofHandler, u::AbstractVector{T}, fieldname
         # VTK output of solution field (or L2 projected scalar data)
         n_c = n_components(ip)
         vtk_dim = n_c == 2 ? 3 : n_c # VTK wants vectors padded to 3D
-        data = fill(NaN * zero(T), vtk_dim, getnnodes(get_grid(dh)))
+        data = fill(T(NaN) * zero(T), vtk_dim, getnnodes(get_grid(dh)))
```

Comment:
```suggestion
        data = fill(T(NaN), vtk_dim, getnnodes(get_grid(dh)))
```

Reply:
Should I fix this consistently in this PR? Because other places use this.

Reply:
Could do if you want to :)
Just noticed that this was not required after the changes here. 
But it could depend on the resolution of the `Int` discussion below...